### PR TITLE
Updates the readme file

### PR DESCRIPTION
**Pull request by altereagle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+This application should cover the following aspects <br>
+  1. A volume notification for a loud classroom <br>
+  2. A notification to load in at a particular time to tell parents when they need to leave <br>
+  3. Media (Video/Music) integration to keep kids quiet <br>
+  4. Status Information retrieval to tell kids where their parents are <br>
+  5. "Pre" notifications that let you know when someone has arrived and will be looking for you <br>
+  6. Displaying student names, grades <br>
+  7. Vibrating when they need to be notified, like when their parents have arrived <br>
+  8. ATB notifications (When they need to go, if they are waiting for their parents, etc.) <br>
+--9. Extra <br>-----------------------------------------------------------------------------
```
Comment (jbendwick) [inline README.md]: what is this line? why is it there?

Comment (jbendwick) [reply README.md]: extra br

**Comment by lward1** [comment]
some type of title of header

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
-# trinity-carpool
-A carpool management app by the students of Austin's Trinity Episcopal School.
+# Trinity Carpool
+### A carpool management app designed and created by the students of Austin's Trinity Episcopal School. 
+-------------------------------------------------------------------------------
```
Comment (jbendwick) [inline README.md]: why is this line longer? lengthen the others or make this shorter

File: README.md [hunk 3]
```diff
@@ -1,2 +1,24 @@
...
+***We are trying to...***
+<br>Create an app that helps carpool run faster, smoother, and more efficiently. 
+This is so that we get longer recess.
+-------------------------------------
+This checkout system is designed to get kids out of the carpool room 
+*as quickly and effeciently as possible.* For example, if two kids are carpooling 
+together, and are not related, the parent picking them up would say the names of 
+the kids being picked up. Then, the kids will get a notification, and go outside. 
+After the parent has the kids, the app will let you check out. Once you have checked out, 
+the teachers will know you have gone. <br>
```
Comment (ironhorse2023) [inline README.md]: The teachers will know you have left

File: README.md (hunk 2 above)
Comment (altereagle) [reply README.md]: I agree, lets do this!

Comment (cdasch) [reply README.md]: I agree! We should make this shorter. Good idea!

File: README.md (hunk 3 above)
Comment (altereagle) [reply README.md]: I like this, this is a good idea! Also should we capitalize the first words?

File: README.md (hunk 1 above)
Comment (mikailaulmer) [inline README.md]: We need to delete this

Comment (mikailaulmer) [reply README.md]: I've deleted it on C9 but I just need to push changes

Comment (altereagle) [reply README.md]: I don't know.

Comment (lward1) [reply README.md]: whats it doing?

**Comment by lward1** [comment]
please revise line 7

File: README.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
 ### A carpool management app designed and created by the students of Austin's Trinity Episcopal School. 
--------------------------------------------------------------------------------
-***We are trying to...***
+-------------------------------------
 <br>Create an app that helps carpool run faster, smoother, and more efficiently. 
-This is so that we get longer recess.
+This is so that we get longer recess.<br>
 -------------------------------------
-This checkout system is designed to get kids out of the carpool room 
-*as quickly and effeciently as possible.* For example, if two kids are carpooling
```
Comment (ironhorse2023) [inline README.md]: …possible.* <br>
For example…

File: README.md [hunk 5]
```diff
@@ -1,2 +1,24 @@
...
+**This application should cover the following aspects...** <br>
+  1. A volume notification for a loud classroom <br>
+  2. A notification to load in at a particular time to tell parents when they need to leave <br>
+  3. Media (Video/Music) integration to keep kids quiet <br>
+  4. Status Information retrieval to tell kids where their parents are <br>
+  5. "Pre" notifications that let you know when someone has arrived and will be looking for you <br>
+  6. Displaying student names, grades <br>
+  7. It vibrates when they need to be notified, for instance, when their parents have arrived <br>
+  8. ATB notifications (When they need to go, if they are waiting for their parents, etc.) <br>
+
```
Comment (jbendwick) [inline README.md]: is this supposed to be there? does it matter?

Comment (mikailaulmer) [inline README.md]: delete this like

**Comment by lward1** [comment]
this looks great! Good job guys!

**Comment by Mirattu** [comment]
This looks great guys! Great job!

**Pull request merged**